uploader/s3aws.go

```diff
@@ -71,7 +71,6 @@ func newS3AwsUploader(accessKey, secretKey, region, bucket, prefix string, timeo
 
 func (u *s3AwsUploader) Upload(failure complainer.Failure, stdoutURL, stderrURL string) (string, string, error) {
 	buf := bytes.NewBuffer([]byte{})
-	err := u.prefix.Execute(buf, map[string]interface{}{"failure": failure})
```

Comment:
Respectfully, I made the change to fix the unrelated gometalinter failures that occurred.  I'll remove my attempted fix, but you'll need to file an issue/track the fix yourself (this isn't my breakage, and it's clearly preceding me).